parsers/ZA.py

```diff
@@ -0,0 +1,961 @@
+from __future__ import annotations
+
+import datetime
+import logging
+import pprint
+import typing
+from collections import OrderedDict
+
+import arrow
+import requests
```

Comment:
Please import and use only the classes, methods and types needed. It makes things easier to understand.
For example:
```python
from requests import Session, Response
```

Reply:
Tbh I am not really in favor of this convention, but I changed my imports in commit 201c005b280d56fa05a757e8b1d3963e138f05a3.